### DISCUSSION TITLE
Installing new firmware and do I have CRSFShot?

### DISCUSSION THREAD

**Discussion by c0debreaker** [EdgeTX/edgetx#3717] [General]
I'm running EdgeTXT 2.6.0. I turned on my Jumper T16 and went inside the Versions tab via SYS key. I don't see CRSFShot listed in the OPTS. Does it mean that the firmware 2.6.0 I have doesn't have CRSFShot?

If I download and install the latest EdgeTX firmware, will I lose all my configurations on the Jumper T16?

Thanks,
Neil

**Comment by pfeerick** [comment]
I'm pretty sure CRSFShot has been part of EdgeTX since 2.4 (i.e. the first version), so you should have it. 

If you're on 2.6, you've already done the hard part of migrating any widgets or screens over, it should just be a matter of updating the firmware, and the SD card pack if you want updated widgets, etc. As with any update, make sure you backup your settings first - either read the radio with Companion and save the `.etx` file as a backup, or make a copy of the RADIO and MODELS folders on the radio SD card...

**Reply by c0debreaker** [reply]
I'm good now. I've been using EdgeTX 2.8 for 2 weeks now and I have not experience any failsafes yet. Thanks a lot!
